lib/reduxable_plug/app.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ReduxablePlug.App do
+  use Application
+
+  def start(_type, _args) do
+    import Supervisor.Spec
+
+    children = [
+      supervisor(Task.Supervisor, [[name: ReduxablePlug.BackgroundJob.supervisor_name()]])
+    ]
+
+    opts = [strategy: :one_for_one, name: ReduxablePlug.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+end
```

Comment:
This is called when the app starts. It starts up a supervisor that monitors our background jobs